src/settings/LostItemFeePolicy/components/ViewSections/LostItemFeeSection/LostItemFeeSection.js

```diff
@@ -44,8 +44,12 @@ const LostItemFeeSection = (props) => {
     : '-';
 
   return (
-    <div data-test-lost-item-policy-detail-fee-section>
+    <div
+      data-testid="viewLostItemFeeSection"
+      data-test-lost-item-policy-detail-fee-section
+    >
       <Accordion
+        data-testid="viewLostItemFeeSectionAccordion"
         id="viewLostItemFeeSection"
         label={<FormattedMessage id="ui-circulation.settings.lostItemFee.lostItemSection" />}
         open={lostItemFeeSectionOpen}
```

Comment:
Should we add test for section below (data-test-item-aged, data-test-patron-billed, data-test-item-recalled-aged, data-test-patron-billed-recalled, data-test-lost-item-fee, data-test-close-loan-after, data-test-replacement-fee)?

Reply:
Tests were added